scripts/just.config.ts

```diff
@@ -125,6 +125,18 @@ export function preset() {
   );
 
   task('screener', screener);
+
+  task('check-if-package-affected', () => {
```

Comment:
nit but IMO there's not a good reason to have this as a just task, you could do it as a command line node script